src/Features/ERDDAP/Platform/Observations/CurrentConditions/index.tsx

```diff
@@ -1,178 +1,45 @@
-import * as React from "react"
-import { Link } from "react-router-dom"
-import { Alert, Col, Row } from "reactstrap"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)